### PR TITLE
Review added doc strings

### PR DESCRIPTION
**Pull request by rosheen33**

### CODE REVIEW

File: common/lib/nodebb_client/categories.py [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 
 class ForumCategory(Category):
+    """
+    Added custom methods to the default Category class of pynodebb package
+    """
 
     def create(self, name, hidden=1, uid=1, **kwargs):
         payload = {'name': name, '_uid': uid, 'hidden': hidden}
```
Comment (muhammadnadeem) [inline common/lib/nodebb_client/categories.py]: please add description for this method too

File: common/lib/nodebb_client/users.py [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 
 class ForumUser(User):
+    """
+    Added custom methods to the default User class of pynodebb package
+    """
+
     def join(self, group_name, user_name, uid=1, **kwargs):
         payload = {'name': group_name, 'username': user_name, '_uid': uid}
```
Comment (muhammadnadeem) [inline common/lib/nodebb_client/users.py]: please add description for this method too

File: lms/djangoapps/faq/models.py [hunk 3]
```diff
@@ -4,6 +4,9 @@
 
 
 class Faq(models.Model):
+    """
+    Model that has the ability to save the data of FAQ page dynamically. We can change its content through admin panel
```
Comment (muhammadnadeem) [inline lms/djangoapps/faq/models.py]: please make this line short if possible

File: lms/djangoapps/homepage/views.py [hunk 4]
```diff
@@ -8,6 +8,9 @@
 
 
 def home_page(request, user=AnonymousUser()):
+    """
+    Override the default homepage for students. Display featured courses and communities
```
Comment (muhammadnadeem) [inline lms/djangoapps/homepage/views.py]: I think this line should be updated too, like "Override default homepage for students" 

No need to tell what content this home page contains

**Comment by rosheen33** [comment]
@muhammadnadeem Kindly review again

**Comment by muhammadnadeem** [comment]
👍    go for merge

**Pull request merged**